pkg/controllermanager/controller/add.go

```diff
@@ -113,6 +114,12 @@ func AddToManager(ctx context.Context, mgr manager.Manager, cfg *controllermanag
 		return fmt.Errorf("failed adding SecretBinding controller: %w", err)
 	}
 
+	if err := (&shootstate.Reconciler{
+		Config: *cfg.Controllers.ShootState,
+	}).AddToManager(mgr); err != nil {
+		return fmt.Errorf("failed adding shootState finalizer reconciler: %w", err)
```

Comment:
For consistency with the other AddToManager calls for the other controllers
```suggestion
		return fmt.Errorf("failed adding ShootState finalizer controller: %w", err)
```

or

```suggestion
		return fmt.Errorf("failed adding ShootState controller: %w", err)
```

Reply:
Thanks for catching this mismatched wording, fixed in my latest updates. Decided to go with the second variant.